### ISSUE TITLE
"Level of Care (Setting)" Settings Should Be Queryable As Folders

### ISSUE THREAD

**Issue by markabajian** [shyamvis/covid-phenotyping#27]
In the April 15th release of the COVID Ontology, we have a "Level of Care (Setting)" tree. And inside that tree are four "settings" where care is delivered: `Ambulatory`, `Emergency`, `Inpatient (Critical Care)`, and `Inpatient (Routine)`.

The four settings have visual attributes of a `Collection`, but because of that, we can't drag the care setting into the query window.  

I feel that these should be set to `Folder` instead, so they can be dragged.

This may already be changed in Version 3 of the ontology, but I am submitting this ticket just in case.  Thank you!

**Comment by markabajian** [comment]
Okay, so I see now that issue #26 and this issue are linked.  And that issue #26 would never have shown up if I did not monkey with the folders mentioned here.  In order to get them to be queryable, I ran the following SQL in my metadata database:
`update [metadata].[dbo].[ACT_COVID]
	set C_VISUALATTRIBUTES = 'FA '
	where C_VISUALATTRIBUTES like 'C%'
	and C_FULLNAME like '%SNOMED%'
;
`
So, yes, now I can query by the folder (this resolves #27) but this also causes the problem that I reported in #26.  If the folder were not queryable, then #26 would not be an issue.

I understand that when we have leaves that rely on queries going to different target tables (**`visit_dimension`** versus **`observation_fact`**) then they cannot really go into the same **folder**... they can only go into a **collection**.  

I hope there can be some resolution for this.  Thank you.
